actions.yaml

```diff
@@ -2,6 +2,12 @@ restart:
   description: Wait for all jobs to be completed and restart jenkins.
 upgrade:
   description: Upgrade jenkins package when using bundle.
+install-dependencies:
+  description: Install Jenkins dependencies for the specified Jenkins version (or the currently installed/ anticipated to be installed version if the Jenkins version is not specified).
```

Comment:
This was primarily needed for the integration tests. I needed a hook that would be executed by the charm to clean up dependencies that are no longer needed and install the new dependencies for a given Jenkins version. I wouldn't expect it to be used by people deploying the charm, although it could be helpful for upgrading or installing specific Jenkins versions. If there is a better way it would be great to know, another option is to include some wording in the description although that seems superfluous since users would just ignore the action if they are not interested in it. The next commit will add that and also limit this file to 100 characters in line length.

Reply:
Make sense to me, thanks, and thanks for adding a note about that in the description.